libc/src/__support/FPUtil/ManipulationFunctions.h

```diff
@@ -230,6 +230,27 @@ LIBC_INLINE T nextafter(T from, U to) {
   return from_bits.get_val();
 }
 
+template <typename T, cpp::enable_if_t<cpp::is_floating_point_v<T>, int> = 0>
+LIBC_INLINE T nextupdown(T x, Sign sign) {
```

Comment:
Use the second template parameter for the `sign`, and use `enable_if_t` on the return type instead.
Also mark this function `constexpr` if possible.